packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2399,7 +2403,9 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// `children` to it.
   ///
   /// Subclasses can override this method to add additional [SemanticsNode]s
-  /// to the tree.
+  /// to the tree. If new [SemanticsNode]s are instantiated in this method
+  /// you'll also have to override [clearSemantics] and dispose those nodes
```

Comment:
`you'll .. there` -> `they must be disposed in [clearSemantics]`. We prefer the passive voice, because the reader may not be the person who wrote the code.